frontend/src/views/AngularWorkflow.jsx

```diff
@@ -17603,6 +17642,19 @@ const AngularWorkflow = (defaultprops) => {
 		</div>
 
 	const changeActionParameterCodeMirror = (event, count, data, actionlist) => {
```

Comment:
when Submit button is hit on the code Editor this function is called as you can see I am trying to update the state of workflow. Even though the state has been updated it is not being reflected. though after you save and refresh the state has been modified.


also selectedAction appears to be empty.